SWIPs/swip-bzz-handshake.md

```diff
@@ -0,0 +1,78 @@
+bzz
```

Comment:
Missing prelude

Reply:
Would be good to signal how ready it is, i.e. spec state likely to be preliminary/first solid draft for a while